train.py

```diff
@@ -153,6 +165,22 @@
     print("=" * 10, env_id, "=" * 10)
     print(f"Seed: {args.seed}")
 
+    if args.track:
+        import wandb
```

Comment:
Please do a try/except with an informative message ;)

Reply:
Are you saying just to print out "tracking this experiment with Weights and Biases"? What exception are you anticipating with try/except.

Reply:
`except ImportError` "if you want to use W&B tracking, please install ...